qa/L0_perf_analyzer/test.sh

```diff
@@ -639,6 +639,29 @@ for PROTOCOL in grpc http; do
         fi
         set -e
     done
+
+    # Testing that trace logging works
+    set +e
+    TRACE_FILE="trace.json"
+    rm ${TRACE_FILE}*
+    $PERF_ANALYZER -v -i $PROTOCOL -m simple_savedmodel_sequence_object -p 2000 -t5 --sync \
```

Comment:
Having an extra block for something like this after David Y's changes would be a nice sanity check, and automatic reminder to test it if it gets re-enabled in the future:
```bash
${PERF_ANALYZER} ... --trace-file trace.json ...
if [ $? -eq 0]; then
  cat ${CLIENT_LOG}
  echo -e "\n***\n*** Test Failed. Expected to fail for unknown arg --trace-file"
  RET=1
fi
```

Reply:
@dyastremsky

Reply:
Nice idea--added